tools/pot/setup.py

```diff
@@ -15,18 +14,23 @@
     long_description = fh.read()
 
 
-def read(*parts):
-    with codecs.open(os.path.join(here, *parts), 'r') as fp:
-        return fp.read()
+def generate_pot_version():
```

Comment:
I am not ok with code copy paste but if there is no way how to avoid it in such a case, let's merge it.